### PR TITLE
feat(command): support multi file paths EE-782

### PR DESCRIPTION
**Pull request by ArrisLee**
Related changes to EE-782

### CODE REVIEW

File: wrapper.go [hunk 1]
```diff
@@ -76,19 +77,24 @@ type composeCommand struct {
 	args    []string
 }
 
-func newCommand(command []string, filePath string) composeCommand {
+func newCommand(command []string, filePaths []string) composeCommand {
+	var args []string
+	for i := range filePaths {
```
Comment (DimaSalakhov) [inline wrapper.go:82]: may use `for _, path := range filePaths` instead

File: wrapper.go [hunk 2]
```diff
@@ -28,12 +29,12 @@ func NewComposeWrapper(binaryPath string) (*ComposeWrapper, error) {
 }
 
 // Up create and start containers
-func (wrapper *ComposeWrapper) Up(filePath, url, projectName, envFilePath, configPath string) ([]byte, error) {
+func (wrapper *ComposeWrapper) Up(filePath []string, url, projectName, envFilePath, configPath string) ([]byte, error) {
```
Comment (DimaSalakhov) [inline wrapper.go:32]:
```suggestion
func (wrapper *ComposeWrapper) Up(filePaths []string, url, projectName, envFilePath, configPath string) ([]byte, error) {
```

File: wrapper.go [hunk 3]
```diff
@@ -28,12 +29,12 @@ func NewComposeWrapper(binaryPath string) (*ComposeWrapper, error) {
...
 	return wrapper.Command(newUpCommand(filePath), url, projectName, envFilePath, configPath)
 }
 
 // Down stop and remove containers
-func (wrapper *ComposeWrapper) Down(filePath, url, projectName string) ([]byte, error) {
+func (wrapper *ComposeWrapper) Down(filePath []string, url, projectName string) ([]byte, error) {
```
Comment (DimaSalakhov) [inline wrapper.go:37]:
```suggestion
func (wrapper *ComposeWrapper) Down(filePaths []string, url, projectName string) ([]byte, error) {
```

**Review by DimaSalakhov — COMMENTED** [review]
LGTM

File: wrapper_test.go [hunk 4]
```diff
@@ -19,40 +20,77 @@ func setup(t *testing.T) *ComposeWrapper {
...
-	_, err = w.Down(filePath, "", "test1")
+	_, err = w.Down([]string{filePathOriginal, filePathOverride}, "", "test1")
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	if containerExists(composedContainerName) {
-		t.Fatal("container should be removed")
+	if containerExists(composeContainerName) {
+		t.Fatal("container should exist")
```
Comment (DimaSalakhov) [inline wrapper_test.go:93]: i believe the old comment was correct: if a container exists, then it's an error

**Review by DimaSalakhov — APPROVED** [review]
all good, just need to revert the error message

File: wrapper_test.go (hunk 4 above)
Comment (ArrisLee) [reply wrapper_test.go]: unexpected change, thanks for the catch

**Pull request merged**